mountainduck/sync.md

```diff
@@ -92,27 +92,27 @@ The file is only saved in a local temporary location and never synced to the ser
 Please make sure to enable the Mountain Duck [Integration](interface.md#context-menu-in-finder-and-windows-file-explorer) in *System Preferences → Extensions → Finder Extensions*.
 ```
 
-## Select Files and Folders to Always Keep Offline
+## Keep Offline
 
-You can choose to make files and folders available offline in the context menu. Those files and folders are located within an obfuscated synchronization cache and can't be accessed outside of your bookmark. You need an active connection to the bookmark to see the offline folders. That means that you have to connect to your server once while you have access to an active internet connection. If you don't disconnect the bookmark and the connection to the server isn't interrupted otherwise you can still access the files that are located within the local synchronization cache. As soon as you are connected to the internet again, all changes are transferred to the server in the background.
+Choose *Mountain Duck → Keep Offline on Local Disk* to make files and folders available offline. The status of the file will change to *In Sync*. The action is recursive for all contained files when a folder is selected and applies to new files found on the remote storage.
 
 ```{image} _images/Sync_Context_Menu_macOS.png
 :alt: Send Command
 :width: 500px
 ```
 
-To reach the context menu right-click on a file or folder in File Explorer (Windows) or Finder (macOS). Refer to [Finder Extension & Windows File Explorer Extension](interface.md#context-menu-in-finder-and-windows-file-explorer).
-
-### Keep Offline
+```{note}
+Files marked with a green checkmark remain accessible, regardless of the network connection to the server, while the volume is mounted. Changes are synchronized in the background when the server is reachable again. This is made possible by caching file contents in an obfuscated cache on local, which is unavailable for direct use.
```

Comment:
How about this?
"As long as the volume is mounted, files marked with a green checkmark remain accessible even if the network connection drops. Changes are synchronized in the background when the server is reachable again. This is made possible by caching file contents in an obfuscated cache on a local disk, which is unavailable for direct use."